robottelo/issue_handlers/bugzilla.py

```diff
@@ -0,0 +1,259 @@
+import re
+import logging
+
+import httpx
+import pytest
+
+from tenacity import retry, stop_after_attempt, wait_fixed
+
+from robottelo.config import settings
+from robottelo.config.base import ImproperlyConfigured
+from robottelo.constants import (
+    CLOSED_STATUSES,
+    OPEN_STATUSES,
+    WONTFIX_RESOLUTIONS
+)
+
+LOGGER = logging.getLogger(__name__)
+
+# match any version as in `sat-6.2.x` or `sat-6.2.0` or `6.2.9`
+# The .version group being a `d.d` string that can be casted to float()
+VERSION_RE = re.compile(r'(?:sat-)*?(?P<version>\d\.\d)\.\w*')
+
+
+def is_open_bz(issue, data=None):
+    """Check if specific BZ is open consulting a cached `data` dict or
+    calling Bugzilla REST API.
+
+    https://hackmd.io/01mIqiG6TOKTrPpTk4jejg?view#Is-Bug-Open
```

Comment:
Going forward, can we have this information in `docs` instead of external resource?